resources/js/components/Litter/AddTags.vue

```diff
@@ -543,6 +576,7 @@ export default {
          */
         async submit ()
         {
+            // this.previousTags = (this.$store.state.litter.tags[this.id] || {});
```

Comment:
Do we need this or not? If its not needed it can be deleted

Reply:
Yes we do. When you click the submit button, then it stores the tags you used.

Reply:
Is it meant to be commented out?

Reply:
I got rid of it. My bad.